protokube/cmd/protokube/main.go

```diff
@@ -188,7 +199,7 @@ func run() error {
 	}
 
 	rootfs := "/"
-	if containerized {
+	if containerized && cloud != "vsphere" {
```

Comment:
I don't know what is 'containerized' getting used for. But the problem with setting root file system as '/rootfs', instead of '/' is that protokube code appends /rootfs to every path. Eg: /rootfs/mnt/master-01 and then protokube run fails saying that /rootfs/mnt/master-01 does not exist or it will keep retrying for some paths to get created/mounted. 

I will try to find out the real purpose of rootfs though.